### PR TITLE
[Flare] Update event component displayName

### PR DESCRIPTION
**Pull request by trueadm**
This is for https://github.com/facebook/react/pull/15927 (if it lands). It updates the `displayName` location on event components.

### CODE REVIEW

File: src/backend/renderer.js [hunk 1]
```diff
@@ -479,7 +479,7 @@ export function attach(
       case IndeterminateComponent:
         return getDisplayName(resolvedType);
       case EventComponent:
-        return type.displayName || 'EventComponent';
+        return type.responder.displayName || 'EventComponent';
```
Comment (bvaughn) [inline src/backend/renderer.js]: Will `type` always have a `responder` value or should we add a null check here? (Will this break FB5 if the DT sync lands before React?)

Comment (trueadm) [reply src/backend/renderer.js]: It will always have a responder.

**Comment by trueadm** [comment]
This might not be needed, as I plan on possibly changing the signature again in React core to allow for a different structure than can be minified better.

**Comment by bvaughn** [comment]
lol

Your comment has been edited from:
> This might not be needed, as I plan on possibly changing the signature again in React core to allow for a different structure than can be minified better.

To:
> It would be good to get this in sometime :)

**Pull request merged**